compiler/rustc_mir_transform/src/coverage/spans.rs

```diff
@@ -393,27 +393,25 @@ impl<'a> CoverageSpansGenerator<'a> {
     }
 
     fn check_invoked_macro_name_span(&mut self) {
-        if let Some(visible_macro) = self.curr().visible_macro(self.body_span) {
-            if !self
-                .prev_expn_span
-                .is_some_and(|prev_expn_span| self.curr().expn_span.ctxt() == prev_expn_span.ctxt())
-            {
-                let merged_prefix_len = self.curr_original_span.lo() - self.curr().span.lo();
-                let after_macro_bang =
-                    merged_prefix_len + BytePos(visible_macro.as_str().len() as u32 + 1);
-                let mut macro_name_cov = self.curr().clone();
-                self.curr_mut().span =
-                    self.curr().span.with_lo(self.curr().span.lo() + after_macro_bang);
-                macro_name_cov.span =
-                    macro_name_cov.span.with_hi(macro_name_cov.span.lo() + after_macro_bang);
-                debug!(
-                    "  and curr starts a new macro expansion, so add a new span just for \
-                            the macro `{}!`, new span={:?}",
-                    visible_macro, macro_name_cov
-                );
-                self.push_refined_span(macro_name_cov);
-            }
+        let Some(visible_macro) = self.curr().visible_macro(self.body_span) else { return };
+        if (self.prev_expn_span)
```

Comment:
Why the extra parentheses?

Reply:
Solely to convince `rustfmt` to keep this subexpression on one line instead of splitting it across 2–3 lines.

(Normally I wouldn't bother, but in this case it seemed silly to take up so many lines on the unimportant bits.)

Reply:
I've pushed a version that doesn't bother with the extra parentheses.

Reply:
Actually I can probably get nicer results with `if let`; I'll try that out and see how it looks.